Error when compiling with OpenJDK 23

#### Checklist

- [x] I checked other issues already, but found no answer/solution
- [x] I checked the documentation and wiki, but found no answer/solution
- [x] I am running the latest version and the issue still occurs
- [x] I am sure that this issue is about SteVe (and not about the charging station software or something unrelated to SteVe)

#### Specifications
```
SteVe Version     : 3.7.1 but also happens with current Git version
Operating system  : Arch Linux
JDK               : OpenJDK 23
Database          : MariaDB 11.x (not part of the issue)
```

#### Expected Behavior

./mvnw package produces an executable version of SteVe

#### Actual Behavior

It throws lots of errors all pointing to some missing dependency/declaration like the following:

[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/xxx/Projekte/steve/src/main/java/de/rwth/idsg/steve/Application.java:[41,9] Symbol nicht gefunden
  Symbol: Variable log
  Ort: Klasse de.rwth.idsg.steve.Application
[ERROR] /home/xxx/Projekte/steve/src/main/java/de/rwth/idsg/steve/Application.java:[41,77] Symbol nicht gefunden
  Symbol: Methode getProfile()
  Ort: Variable sc von Typ de.rwth.idsg.steve.SteveConfiguration
...

#### Steps to Reproduce the Problem

  1. Install/activate OpenJDK 23
  2. Clone SteVe
  3. ./mvnw package


#### Additional context

I switched to OpenJDK 21 and that works like it used to. Also, OpenJDK 23 has no issues running the .jar compiled on OpenJDK 21.

--- a/pom.xml
+++ b/pom.xml
@@ -181,6 +181,13 @@
                 <configuration>
                     <source>${java.version}</source>
                     <target>${java.version}</target>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                            <version>${lombok.version}</version>
+                        </path>
+                    </annotationProcessorPaths>
                 </configuration>
             </plugin>
 

